app/client/components/transferAccount/transferAccountManager.jsx

```diff
@@ -35,7 +35,7 @@ class TransferAccountManager extends React.Component {
     this.state = {
         action: 'select',
         transfer_type: 'ALL',
-        create_transfer_type: 'WITHDRAWAL',
+        create_transfer_type: 'RECLAMATION',
```

Comment:
isn't it weird how "reclaim" is spelt with an "i" but "reclamation" isn't?! stoopid english

Reply:
Haha yeah